backend/src/main/java/com/zpi/backend/game/GameService.java

```diff
@@ -81,11 +84,17 @@ public Game getGame(long id) throws GameDoesNotExistException {
         return gameOptional.get();
     }
 
-    public GameDTO getGameDTO(long id) throws GameDoesNotExistException{
-        return new GameDTO(getGame(id));
+    public GameDTO getGameDTO(long id,String language) throws GameDoesNotExistException, BadRequestException {
+        if(language.equals(LanguageCodes.ENGLISH)){
+            return new EnGameDTO(getGame(id));
```

Comment:
the same thing like in Categories, so result DTO should have only name and shortDescription - without pl